yaml.go

```diff
@@ -88,6 +88,7 @@ func JSONToYAML(j []byte) ([]byte, error) {
 	// universally. go-yaml does go through the effort of picking the right
 	// number type, so we can preserve number type throughout this process.
 	err := yaml.Unmarshal(j, &jsonObj)
+	yaml.FutureLineWrap()
```

Comment:
we **definitely** don't want to change global state of the yaml library in this function... it should be up to consumers of this library to decide if they want to flip that flag

Reply:
My bad, I was thinking it was in unit test. of course, not wanted.